packages/manager/modules/pci/src/projects/project/project.controller.js

```diff
@@ -63,6 +63,12 @@ export default class ProjectController {
       });
   }
 
+  get feedbackUrl() {
+    return this.user.ovhSubsidiary.toUpperCase() === 'FR'
```

Comment:
Is it possible to have something like 
```suggestion
return [`__FEEDBACK_URL_${this.user.ovhSubsidiary.toUpperCase()}__`] || __FEEDBACK_URL_EN__
```
and does it make sense

Reply:
Sorry it's not possible. That doesn't return the same value.